src/main/java/frc/robot/subsystems/ElevatorIO.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface ElevatorIO {
+    
+    @AutoLog
+    public static class ElevatorIOInputs {
+
+        public double elevatorPosition = 0.0;
```

Comment:
We're going to track the elevator's height and velocity using a [LaserCAN](https://www.thethriftybot.com/products/pre-order-grapple-robotics-lasercan). This also got used on the end effector, so you can look at #10 to see how to implement it.